@xen-orchestra/lite/src/locales/fr.json

```diff
@@ -36,6 +36,7 @@
   "loading-hosts": "Chargement des hôtes…",
   "log-out": "Se déconnecter",
   "login": "Connexion",
+  "master": "Master",
```

Comment:
No need to use i18n at all if we're gonna keep it as "Master" in every language.